### PR TITLE
Add contacts area section

### PR DESCRIPTION
**Pull request by PNdunguMaina**
In this project I did the following:

- Updated index.html file for the contacts section
- Updated style.css file for the contacts section
- Collaborated with two of my Microverse colleagues on this part of the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -195,6 +195,88 @@ <h4>Multi-Post Stories</h4>
...
+    <section class="contacts-area">
+      <div class="contacts">
+        <h6>CONTACT US</h6>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form class="contacts-form" action="#" method="post">
```
Comment (danmainah) [inline index.html]: - [ ] Kindly complete your action method, and ensure to integrate with [formspree service](https://formspree.io/html).

File: index.html [hunk 2]
```diff
@@ -195,6 +195,88 @@ <h4>Multi-Post Stories</h4>
...
+            <textarea
+              name="msg"
+              id="msg"
+              placeholder="Enter text here"
+            ></textarea>
+          </li>
+        </ul>
+      </form>
+
+      <a class="button button-frame" href="#">see Project</a>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly ensure your button, is a `<button> element or an `<input>` element , if you choose `<input>` element ensure  you add type `submit`.Also, ensure the hover effect is aligned to the design given.

File: index.html [hunk 3]
```diff
@@ -195,6 +195,88 @@ <h4>Multi-Post Stories</h4>
...
+      <form class="contacts-form" action="#" method="post">
+        <ul class="input-area">
+          <li class="text-area">
+            <label for="name"></label>
+            <input
+              type="text"
+              id="name"
+              name="user_name"
+              placeholder="Text Input"
+            />
```
Comment (danmainah) [inline index.html]: - [ ] Kindly add the validation for the name input field, and add the `maxlength="30"` attribute, which checks the length of the input and ensures it's not beyond 30 characters.

File: index.html [hunk 4]
```diff
@@ -195,6 +195,88 @@ <h4>Multi-Post Stories</h4>
...
+              placeholder="jeremy@gmail.com"
+            />
+          </li>
+
+          <li class="message-area">
+            <textarea
+              name="msg"
+              id="msg"
+              placeholder="Enter text here"
+            ></textarea>
```
Comment (danmainah) [inline index.html]: - [ ]  Kindly add the validation for the message input field, and add the `maxlength="500"` attribute, which checks the length of the input and ensures it's not beyond 500 characters.

File: index.html (hunk 1 above)
Comment (danmainah) [reply index.html]: - [ ] Kindy ensure all input fields have the `required` attribute to activate the validation on submit.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey  @PNdunguMaina 😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

[Optional] All fields must be `required`. Please add `required` attribute in all input fields 👍🏻 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — COMMENTED** [review]
Hi @PNdunguMaina,

## Wow, you did it 🎉 
Brilliant

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To highlight

- Last week's reviewers' suggestions were implemented✔️ 
- The app works well✔️ 
- The form works✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**